src/sources/aws_sqs/source.rs

```diff
@@ -90,35 +90,33 @@ impl SqsSource {
         };
 
         if let Some(messages) = receive_message_output.messages {
-            let mut receipts_to_ack = vec![];
+            let mut receipts_to_ack = Vec::with_capacity(messages.len());
+            let mut events = Vec::new();
```

Comment:
`events` should probably also be pre-allocated with `Vec::with_capacity(messages.len())`
While there could be more events, the most likely case is usually 1 event per message.